src/Console/Migration/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
How about moving migrations out of `src/Console`? These could be treated as non-source i.e. be in the `migrations` at the top level or at least could be moved out of `Console`.

Reply:
Maybe you're right. In addition, I also wanted to make a drop cache when applying / rolling back migrations

Reply:
About the location of the migrations:
Migrations change (add) with Entities. I think they have the right to be in Src along with Entities

Reply:
OK but, I think, not in Console.

Reply:
`src/Migration` ?